addons/l10n_th/views/report_cash_voucher_templates.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_cash_voucher_document" inherit_id="account.report_payment_receipt_document" primary="True">
+        <xpath expr="//h3" position="before">
+            <!-- Add Company Information -->
+            <t t-set="comp" t-value="o.company_id.partner_id"/>
+            <div class="row">
+                <div class="col-6 offset-6">
+                    <div>
+                        <span t-field="comp.name" /><br/>
+                        <span t-field="comp.vat"/> <span t-esc="comp.l10n_th_branch_number if comp.l10n_th_company_info == 'branch' else 'Headquarter'" /><br/>
+                        <span t-esc="comp"
+                            t-options='{"widget": "contact", "fields": ["address", "phone"], "no_marker": True, "phone_icons": True}'/>
+                    </div>
+                </div>
+            </div>
+        </xpath>
+        <xpath expr="//h3" position="replace">
+            <h3 class="pt-2"><strong><span t-if="o.payment_type == 'inbound'">Receipt Voucher</span><span t-else="">Payment Voucher</span>: <span t-field="o.name" /></strong></h3>
+        </xpath>
+        <xpath expr="//table" position="replace">
+            <table class="table table-sm">
+                <thead>
+                    <tr>
+                        <th><span>Account</span></th>
+                        <th><span>Label</span></th>
+                        <th><span>Debit</span></th>
+                        <th><span>Credit</span></th>
+                        <th><span>Analytic Account</span></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <t t-set="total_debit" t-value="0"/>
+                    <t t-set="total_credit" t-value="0"/>
+                    <t t-foreach="o.move_id._get_reconciled_invoices_partials()" t-as="rec">
```

Comment:
How is it functionally that much different than the standard payment receipt report that we need to completely replace the table?  (why can not we do the same?)

Reply:
The requirement can be found here https://www.odoo.com/web#id=2879718&cids=5&menu_id=4720&action=333&active_id=1822&model=project.task&view_type=form section 1.3.3


Although it extracts the information the same way at the beginning, the content of the table is different. For the cash voucher we take the journal items that are being reconciled by this payment, while the original payment receipt is for the journal entries. 

At the same time, since the table header and body content is completely different anyway, I figured it'd be much simpler to just replace the whole table altogether